src/homoscedasticity.jl

```diff
@@ -0,0 +1,79 @@
+# homoscedasticity.jl
```

Comment:
Please use the same header as other files.

Reply:
By this, do you mean use a proper name for the test? I am not sure it has one but I will try to find out.

Reply:
Sorry I hadn't realized files in this package use this header. We don't repeat the file name and the full license in other packages.